pom.xml

```diff
@@ -464,7 +464,7 @@
 					<version>3.0.1</version>
 					<configuration>
 						<excludes>
-							<exclude>org/apache/pirk/benchmark/**</exclude>
+							<exclude>org/apache/pirk/test/benchmark/**</exclude>
```

Comment:
Don't think this is needed, the test/ folder is never packaged into Maven artifacts.